<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YahooFinance</class>
 <widget class="QMainWindow" name="YahooFinance">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>755</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Yahoo Finance Scraper</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(200, 174, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Logo (1).png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
      </property>
      <property name="text">
       <string>Get stock information</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Number Of Stocks: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="NumberOfStocksCB">
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QProgressBar" name="progressBar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
}

QProgressBar::chunk {
	background-color: rgb(200, 174, 255);
    width: 20px;
}</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="progress_msg">
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 0, 127);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_5">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Select information to show :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="informationSelectCB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Show all stocks</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Top estimated profitable stocks</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Top earning per share stocks</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Top dividend stocks</string>
         </property>
        </item>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QPushButton" name="showInfoBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Show</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item alignment="Qt::AlignLeft">
     <widget class="QPushButton" name="ExportBtn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Export to excel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: 6pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string> Â© Dvir.b.s, Arik.z, Shenhav.h</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>21</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
